src/core/namespace.c

```diff
@@ -1404,6 +1470,13 @@ static int mount_image(
         if (r < 0)
                 return log_debug_errno(r, "Failed to mount image %s on %s: %m", mount_entry_source(m), mount_entry_path(m));
 
+        if (m->mode == MOUNT_EXTENSION_IMAGE && verity.designator != _PARTITION_DESIGNATOR_INVALID) {
```

Comment:
hmm, the part right of the && should probably be: `verity_settings_set(&verity)`